x-pack/plugins/ml/public/application/explorer/explorer_charts/components/explorer_chart_label/entity_filter/entity_filter.tsx

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { FC } from 'react';
+import { EuiButtonIcon, EuiToolTip } from '@elastic/eui';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { i18n } from '@kbn/i18n';
+import {
+  ENTITY_FIELD_OPERATIONS,
+  EntityFieldOperations,
+} from '../../../../../../../common/util/anomaly_utils';
+import './entity_filter.scss';
+
+interface EntityFilterProps {
+  onFilter: (params: {
+    influencerFieldName: string;
+    influencerFieldValue: string;
+    operation: EntityFieldOperations;
+  }) => void;
+  influencerFieldName: string;
+  influencerFieldValue: string;
+}
+export const EntityFilter: FC<EntityFilterProps> = ({
```

Comment:
The filter icons from the Top Influencers list and Anomalies table are no longer working for me. I see these errors in the console:

![image](https://user-images.githubusercontent.com/7405507/112325680-d1145900-8cab-11eb-8b5a-5208c2c526be.png)

Reply:
Thanks for catching that - this should be fixed now [`dbe8cee` (#94396)](https://github.com/elastic/kibana/pull/94396/commits/dbe8cee568eb90f7148c748f506640a7494a0b83)